src/select.ts

```diff
@@ -197,9 +201,18 @@ export class PrismaSelect {
   private filterBy(modelName: string, selectObject: any, isRootList?: boolean) {
     const model = this.model(modelName)
     if (model && typeof selectObject === 'object') {
+      let defaultFields = {}
```

Comment:
I think it's better to use a different name, to reduce confusion with `this.defaultFields`

```suggestion
      let select = {}
```

Reply:
resolved, thanks